docarray/array/storage/weaviate/find.py

```diff
@@ -113,6 +145,10 @@ def _filter(
 
             doc.tags['wid'] = result['_additional']['id']
 
+            if additional:
+                for add in additional:
```

Comment:
if additional is a dictionary, why is value never used?

Reply:
The value is passed from the results directly into a tag using the key
```python
            if additional:
                for add in additional:
                    doc.tags[f'{add}'] = result['_additional'][add] #<- the value is set as a tag by the key
```